Dan Tudor/04-3.13/main.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+    int s[100],i,n,x,pos=-1;
+    scanf("%d%d",&n,&x);
```

Comment:
because you have no delimiters between them, am I right in assuming the first number can only be of 1 digit maximum?